crates/reporting/src/error/canonicalize.rs

```diff
@@ -1346,6 +1346,32 @@ pub fn can_problem<'b>(
             doc = report.doc;
             title = report.title;
         }
+
+        Problem::ReturnOutsideOfFunction { region } => {
+            doc = alloc.stack([
+                alloc.concat([
+                    alloc.reflow("This "),
+                    alloc.keyword("return"),
+                    alloc.reflow(" statement doesn't belong to a function:"),
+                ]),
+                alloc.region(lines.convert_region(region), severity),
+            ]);
+
+            title = "RETURN OUTSIDE OF FUNCTION".to_string();
+        }
+
+        Problem::StatementsAfterReturn { region } => {
+            doc = alloc.stack([
+                alloc.concat([
+                    alloc.reflow("This code won't run because it follows a "),
+                    alloc.keyword("return"),
+                    alloc.reflow(" statement:"),
+                ]),
+                alloc.region(lines.convert_region(region), severity),
+            ]);
```

Comment:
Usually our errors look like:

```
you did X

    code doing X

why you can't do X and/or help.
```

I'd add a line below the code like:
```
Hint: Did you forget to add a condition?
```

It doesn't have to be that specifically, but something like it so we keep the style consistent.

Reply:
Good suggestion!